functions/dialogflow/functions/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Not needed, `node_modules` is excluded in the top level gitignore.

Reply:
Removed.